packit/utils/local_test_utils.py

```diff
@@ -0,0 +1,93 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os
+import re
+from pathlib import Path
+from typing import (
+    Optional,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class LocalTestUtils:
+    @staticmethod
+    def _build_tmt_cmd(
```

Comment:
Probably part of the other comments about splitting this feature to a different PR.

But to quickly note something, the `LocalTestUtils` has no meaning when all the functions are static. It should be plain functions instead. Use classes if:
- You want to inherit and override some implementations (in which case `classmethod` is a more pythonic elegant way of designing it)
- You need to instantiate an object with different parameters (in which case you use plain methods)